cmake/platform/VxWorks.cmake

```diff
@@ -6,4 +6,4 @@ add_definitions(-D_WRS_KERNEL)
 set(FPRIME_USE_BAREMETAL_SCHEDULE OFF)
 
 # Add VxWorks specific headers into the system
-include_directories(SYSTEM "${FPRIME_FRAMEWORK_PATH}/Fw/Types/VxWorks")
+include_directories(SYSTEM "${CMAKE_CURRENT_LIST_DIR}/types")
```

Comment:
Add directory for Vx6 and Vx7. Move Vx7 types to Vx7 and copy Vx7 types to Vx6. This "Vx6" types file will be edited in a separate PR.